tests/test_path_feedback.py

```
@@ -0,0 +1,150 @@
+def test_missing_full_path_stays_a_single_line(no_workspace, tmp_path):
+    target = tmp_path / "absent.mp4"
+
+    message = video.describe_missing_input(str(target))
+
+    assert message == f"no such file: {target}"
+
+
+def test_every_reader_raises_the_explaining_message(no_workspace, monkeypatch, tmp_path):
+    monkeypatch.chdir(tmp_path)
+
+    with pytest.raises(FileNotFoundError, match="no such file: hello.mp4"):
```

Ruff (0.16.3)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)